go.mod

```diff
@@ -86,3 +86,5 @@ require (
 	google.golang.org/protobuf v1.31.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/maas/gomaasclient v0.1.0 => github.com/skatsaounis/gomaasclient v0.0.0-20231225220956-4c6bcbe1988d
```

Comment:
Is this meant to still be here?

Reply:
It will be removed when https://github.com/maas/gomaasclient/pull/42 is merged and a new version of gomaasclient includes it. Until then this PR here is blocked.